core/base/imagestreams.yaml

```diff
@@ -17,6 +17,14 @@ spec:
 ---
 apiVersion: image.openshift.io/v1
 kind: ImageStream
+metadata:
```

Comment:
why are we adding investigator image to the core?
is it because its being used by adviser workflow and investigator deploymentconfig?

Reply:
exactly and later solver will use it in one of the task and graph-refresh-job as well :)